code-challenges.test.js

```diff
@@ -21,36 +20,146 @@
 // Example input: 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
+// a describe method that lists a function with expect statements
+
+describe("fibonacciLength", () => {
+  it("takes in a number >2 and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    // expected input: 6, expected output: our example values above
+    // expected input: 10, expected output: our example values above
+
+    expect(fibonacciLength(6)).toEqual([1, 1, 2, 3, 5, 8]);
+    expect(fibonacciLength(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55]);
+  });
+});
+
+// Run file in jest
+// RED: failed test - fibonacciLength is not defined
 
 // b) Create the function that makes the test pass.
 
+// create a function that takes in a num as a parameter
+
+var input1 = 6;
+var input2 = 10;
+
```

Comment:
I don't think you need these test inputs. All the code that exists should be either in your test or in your function.